src/Lachain.Core/RPC/HTTP/Web3/Web3DataFormatUtils.cs

```diff
@@ -29,6 +29,11 @@ public static string Web3Number(ulong x)
         {
             return x.ToHex(false);
         }
+        
+        public static string Web3Number(IEnumerable<byte> x)
+        {
+            return x.ToHex();
```

Comment:
Are you sure it is a correct convertation? It should be similar to the function for UInt256 type, as far as I remember